tests/script/partials/sdk-js-script-tags.js

```diff
@@ -24,11 +24,6 @@ const jaOutput = `<script
   defer
 ></script>`;
 
-it('creates correct tags when no locale specified', () => {
-  const partialUsage = '{{> script/partials/sdk-js-script-tags sdkVersion=1.8}}'
```

Comment:
do we ever have a case where locale is not set (maybe when there's no locale_config.json)?

Reply:
Right now, whenever we invoke this partial, the `locale` is always set. If it's not in the `locale_config` or `global_config`, we simply use `en`.